src/mRides-server/Controllers/UserController.cs

```diff
@@ -47,9 +48,10 @@ public object getUser(int id)
         [HttpPost]
         public object createUser([FromBody]User user1)
         {
-            _userCatalog.createUser(user1);
-            //var r = _context.Users.Include(c => c.RidesAsDriver).Single(u=>u.ID==1);
-            return user1; 
+            //_userCatalog.create(user1);
+            ////var r = _context.Users.Include(c => c.RidesAsDriver).Single(u=>u.ID==1);
+            //return user1;
+            return null; 
```

Comment:
Ohh yes. It's because the code I commented out belongs to user catalog. This needs to be refractored. I will do it